### PR TITLE
Update readme.md to contain nvm instructions.

### PR DESCRIPTION
**Pull request by dPliakos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,18 +13,22 @@ Welcome to Open Source Student Network page repository!
...
+1. Run `nvm use`.
+2. Open development server.
+At the repository directory run `npm run develop`.
 - The page will be available at [localhost:8000](htpp://localhost:8000) by default
 
 ### <a name="build"></a> Build
 
-1. Building the page. At the repository directory run `gatsby build`
-2. Start the server. Run `gatsby build` to initialize a gatsby server at your machine.
+1. Run `nvm use`.
```
Comment (iropal) [inline README.md]: @dPliakos  What do you think about:
"At the repository directory run `nvm use`."

File: README.md [hunk 2]
```diff
@@ -13,18 +13,22 @@ Welcome to Open Source Student Network page repository!
...
-3. Install node dependencies. On repo directory run `npm install`.
+3. Clone the repository. Run `git clone https://github.com/ossn/ossn-frontend.git`.
+4. Install nvm. Follow the instructions on [nvm repo](https://github.com/creationix/nvm#installation).
+5. Run `nvm use`.
+6. Install node dependencies. On repo directory run `npm install`.
 
 ### <a name="develop"></a> Develop
-1. Open development server.
-At the repository directory run `npm run develop`
+1. Run `nvm use`.
```
Comment (iropal) [inline README.md]: @dPliakos  What do you think about:
"At the repository directory run `nvm use`."

File: README.md (hunk 1 above)
Comment (dPliakos) [reply README.md]: It seems better  like this, let me change it

File: README.md [hunk 3]
```diff
@@ -13,18 +13,21 @@ Welcome to Open Source Student Network page repository!
...
+2. Open development server. Run `npm run develop`.
 - The page will be available at [localhost:8000](htpp://localhost:8000) by default
 
 ### <a name="build"></a> Build
 
-1. Building the page. At the repository directory run `gatsby build`
-2. Start the server. Run `gatsby build` to initialize a gatsby server at your machine.
+1. At the repository directory run `nvm use`.
+2. Building the page. Run `gatsby build`.
+3. Start the server. Run `gatsby build` to initialize a gatsby server at your machine.
```
Comment (iropal) [inline README.md]: @dPliakos I guess it is 'gatsby serve', not `gatsby build` :)

Comment (dPliakos) [reply README.md]: Let me review the whole `README.md` then :)

**Pull request merged**